src/components/Alerts/AlertWrapper.ts

```diff
@@ -0,0 +1,116 @@
+import styled, { ThemedStyledProps } from 'styled-components';
+import { AlertType } from './Alerts';
+import { media, ComponentTheme } from '../../theme';
+
+const typeColors = (
+  props: ThemedStyledProps<{ index: number; type: AlertType }, ComponentTheme>,
+): { color: string; background: string } => {
+  switch (props.type) {
+    case AlertType.Danger:
+      return {
+        background: props.theme.alerts.color.dangerSecondary,
```

Comment:
A reason why this isn't coming from a background property?
So `props.theme.alerts.background.danger`.

Reply:
You are right, this was a blind copy from the existing code, i'll change it